technology/java-spring-boot/build.gradle

```diff
@@ -18,25 +17,25 @@ repositories {
 }
 
 dependencies {
+    implementation platform("ai.timefold.solver:timefold-solver-bom:${timefoldVersion}")
+
     implementation "org.springframework.boot:spring-boot-starter-web"
-    implementation "org.springframework.boot:spring-boot-starter-data-rest"
-    implementation "org.springframework.boot:spring-boot-starter-data-jpa"
+    implementation "ai.timefold.solver:timefold-solver-spring-boot-starter"
+
+    implementation "org.springdoc:springdoc-openapi-starter-webmvc-ui:2.1.0"
+
     testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation("ai.timefold.solver:timefold-solver-benchmark")
+    testImplementation("ai.timefold.solver:timefold-solver-test")
     testImplementation "io.rest-assured:rest-assured"
     testImplementation "org.awaitility:awaitility"
 
-    implementation platform("ai.timefold.solver:timefold-solver-bom:${timefoldVersion}")
-    implementation "ai.timefold.solver:timefold-solver-spring-boot-starter"
-    testImplementation("ai.timefold.solver:timefold-solver-test")
-    testImplementation("ai.timefold.solver:timefold-solver-benchmark")
-    testImplementation "org.assertj:assertj-core:${assertjVersion}"
 
-    runtimeOnly "com.h2database:h2"
-    runtimeOnly "org.webjars:webjars-locator:0.37"
+    runtimeOnly "org.webjars:webjars-locator:0.50"
     runtimeOnly "ai.timefold.solver:timefold-solver-webui"
     runtimeOnly "org.webjars:bootstrap:5.2.3"
     runtimeOnly "org.webjars:jquery:3.6.4"
-    runtimeOnly "org.webjars:font-awesome:5.11.2"
+    runtimeOnly "org.webjars:font-awesome:5.15.1"
     runtimeOnly "org.webjars.npm:js-joda:1.11.0"
```

Comment:
If you're upgrading versions of these components, please do it consistently with the other School Timetabling quickstart. We want them to be truly identical, except for the places where they can not be for obvious reasons.

(Please remember to test the quickstart's UI after modifying these dependency versions.)

In general, let's try and keep the versions of *all* dependencies (not just this) in sync across quickstarts. Much of this is done automatically through Dependabot, but for some reason, Gradle doesn't work too well with it.

Reply:
The Gradle files have been updated to match the Maven files, and all modules are now using version `5.15.1` of the `font-awesome` library.

I have successfully executed the Gradle commands to build and run the projects `technology/java-spring-boot` and `use-cases/school-timetabling`.